webapp-django/crashstats/topcrashers/views.py

```diff
@@ -298,20 +209,18 @@ def topcrashers(request, days=None, possible_days=None, default_context=None):
         _range_type=range_type,
     )
 
-    if api_results['total'] > 0:
-        tcbs = api_results['facets']['signature']
-    else:
-        tcbs = []
+    tcbs = api_results
```

Comment:
This assignment is unnecessary.

Also, I'm a little worried about the noun mismatch here; `get_topcrashers_results` returns `SignatureStats` instances, instead of "results" (whatever that means). Then, we call the things inside `tcbs` "crashes", even though they're more stats about crashes.

Should we call these things TopCrashers?